### DISCUSSION TITLE
WebSSH keyboard as separate app

### DISCUSSION THREAD

**Discussion by toledobastos** [isontheline/pro.webssh.net#1242] [Ideas]
This may come as a surprise but WebSSH is the only keyboard in the entire iOS ecosystem that has a functional set of arrow keys. I wish I could use it with other apps as multitouch features don't work too well with terminal prompts. Would it be possible to release this keyboard, specifically the function bar, as a standalone app to be used elsewhere? This would be very useful.

**Comment by isontheline** [comment]
Woohoo thank you @toledobastos 🙏

You are the first to say that 😅
Are you talking about the 4 arrows key that you can swipe?

I don’t know if a standalone keyboard could be a solution because every app need to handle the ANSI Key Codes differently.

**Reply by toledobastos** [reply]
That's right. I've now gone through a rabbit hole of testing every single keyboard app available at the App Store (Gboard, SwiftKey, etc.) None of them have that single key that works as an arrow set by touching and moving it up, down, left, or right. I don't know how difficult it would be to release it as a standalone keyboard app but it can't be too hard because may of the keyboard apps at th App Store feature a function bar just like the one on WebSSH, except they're dedicated to cosmetic features and therefore useless for any serious work.

**Reply by toledobastos** [reply]
By the way there are dozens of threads out there (Reddit, Stackoverflow, etc.) of people asking specifically about this feature. So there certainly is use for something like that.

**Reply by isontheline** [reply]
On my side the arrows directional key send SSH ANSI escape codes, so I don’t know how to move cursor inside a simple text app
